components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -233,11 +234,11 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
         }
         phase = StartPhase.Stopping;
         statusMessage = <div>
-          <div className="flex space-x-3 items-center rounded-xl m-auto px-4 h-16 w-72 mt-4 bg-gray-100">
+          <div className="flex space-x-3 items-center text-left rounded-xl m-auto px-4 h-16 w-72 mt-4 bg-gray-100">
             <div className="rounded-full w-3 h-3 text-sm bg-gitpod-kumquat">&nbsp;</div>
             <div>
               <p className="text-gray-700 font-semibold">{this.state.workspaceInstance.workspaceId}</p>
-              <a href={this.state.workspace?.contextURL}><p className="w-56 truncate hover:underline" >{this.state.workspace?.contextURL}</p></a>
+              <a href={this.state.workspace?.contextURL} target="_blank"><p className="w-56 truncate hover:underline" >{this.state.workspace?.contextURL}</p></a>
```

Comment:
Please no more `target="_blank"`. Users should decide how to open links.

Reply:
Agree, but please see https://github.com/gitpod-io/gitpod/issues/3707

Reply:
Solved with `target="_parent"` -- it makes simple click work as expected (i.e. navigate the parent), and is ignored on Cmd+Click (opens a new tab).